nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/main/java/org/apache/nifi/properties/sensitive/SensitivePropertyMetadata.java

```diff
@@ -14,10 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.properties;
+package org.apache.nifi.properties.sensitive;
 
-public interface SensitivePropertyProviderFactory {
-
-    SensitivePropertyProvider getProvider();
+/**
+ * Parent interface to allow provider-specific encryption metadata around protected property values.
+ */
+public interface SensitivePropertyMetadata {
```

Comment:
We may be able to remove the concept of encryption metadata as the change from a long-lived, multiple key-handling provider implementation to a flyweight, single-use provider might have rendered this unnecessary.

Reply:
Metadata bits removed.